### PR TITLE
chore: api examples to es6

### PR DESCRIPTION
**Pull request by jajugoguma**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
- Change examples in api document(jsdoc) and README to ES6
- Prettier table of contents in README

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

File: README.md [hunk 1]
```diff
@@ -12,17 +12,33 @@
...
-- [Examples](#-examples)
-- [Install](#-install)
-- [Usage](#-usage)
-- [Browser Support](#-browser-support)
-- [Pull Request Steps](#-pull-request-steps)
-- [Contributing](#-contributing)
-- [TOAST UI Family](#-toast-ui-family)
-- [License](#-license)
+- [TOAST UI Component : Context Menu](#toast-ui-component--context-menu)
+  - [🚩 Table of Contents](#-table-of-contents)
```
Comment (jungeun-cho) [inline README.md]: 목차 윗 내용 기재 제거해도되겠네요~

File: src/js/contextmenu.js [hunk 2]
```diff
@@ -44,12 +44,13 @@ const DEFAULT_ZINDEX = 999;
  *     If you do not want to send the hostname, this option set to false.
  * @example
  * //-- #1. Get Module --//
- * var ContextMenu = require('tui-context-menu'); // node, commonjs
- * var ContextMenu = tui.ContextMenu; // distribution file
+ * import ContextMenu from 'tui-context-menu'; // ES6
+ * // const ContextMenu = require('tui-context-menu'); // CommonJS
+ * // const ContextMenu = tui.ContextMenu;
```
Comment (jungeun-cho) [inline src/js/contextmenu.js]: `//` 주석처리가 필요할까요?
네임스페이스 사용하는건 // Browser 라고 명시해도좋을것같아요.

**Review by jungeun-cho — APPROVED** [review]
확인했습니다.

**Pull request merged**